tests/test_basic_ops_dict.py

```diff
@@ -205,6 +204,23 @@ def test_dict_get_with_default(
     assert c.get(key, default_val) == default_val
 
 
+@pytest.mark.parametrize(
+    "d,exc",
+    [
+        ({"hello": "${foo}"}, InterpolationKeyError),
+        (
+            {"hello": "${foo}", "foo": "???"},
+            InterpolationToMissingValueError,
+        ),
+        ({"hello": DictConfig(content="${foo}")}, InterpolationKeyError),
+    ],
+)
+def test_dict_get_with_default_errors(d: Any, exc: type) -> None:
+    c = OmegaConf.create(d)
+    with pytest.raises(exc):
+        c.get("hello", default_value=123)
+
+
```

Comment:
You have lost some coverage.
Previously this was tested with 
```python
@pytest.mark.parametrize("struct", [None, True, False])
@pytest.mark.parametrize("default_val", [4, True, False, None])
```

You can fix by using a class:
```python
@pytest.mark.parametrize("struct", [None, True, False])
@pytest.mark.parametrize("default_val", [4, True, False, None])
class TestGetWithDefault:

  def test_dict_get_with_default(...
    ...

  def test_dict_get_with_default_errors(d: Any, exc: type) -> None:
    ...
```

Reply:
Done in 2f85931